src/flambda_flags

```diff
@@ -0,0 +1 @@
+()
```

Comment:
Please remove this file.

Earlier I hadn't added it to `.gitignore`. Later I added it and changed it to `.flambda_flags`, but it doesn't get deleted automatically if you were already using the old version of the code.